### PR TITLE
cloud-controller-manager should not show empty version(2037680-bz)

### PR DESCRIPTION
**Pull request by miyadav**
@jhou1 @sunzhaohua2 @huali9 , PTAL 

[Output](https://docs.google.com/document/d/13sJ2CDZ6lgvDLjpf4KQMEgRDetubcAXsizOhM4o2Peo/edit?usp=sharing)

### CODE REVIEW

File: features/machine/machine.feature [hunk 1]
```diff
@@ -587,3 +587,11 @@ Feature: Machine features testing
     And the output should contain:
       | Forbidden: publicIP is not allowed in Azure disconnected installation |
 
+  # @author miyadav@redhat.com
+  # @case_id OCP-47658
+  @admin
+  @4.10
+  Scenario: Operator cloud-controller-manager should not show empty version
+    Given I switch to cluster admin pseudo user
+    Then evaluation of `cluster_operator('cloud-controller-manager').versions` is stored in the :version clipboard
```
Comment (jhou1) [inline features/machine/machine.feature:596]: If it is an array, can we please use plural -> `:versions` and `cb.versions[0]`

**Comment by jhou1** [comment]
Added one comment, other than that lgtm

File: features/machine/machine.feature (hunk 1 above)
Comment (miyadav) [reply features/machine/machine.feature]: Thank you , made the change

**Pull request merged**